Packs/MicrosoftExchangeOnline/ReleaseNotes/1_4_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### O365 - Security And Compliance - Content Search v2
+
+- %%UPDATE_RN%%
```

Comment:
Please update the release notes: `Added the ***o365-sc-case-hold-policy-set*** command, which __`.

Reply:
Done